README.md

```diff
@@ -495,17 +495,50 @@ as shown in the examples above.
 
 ## Component Formatting
 
-> NOTE: At this time, we __do not recommend using [`dartfmt`](https://github.com/dart-lang/dart_style)__, as it 
-> _greatly_ decreases the readability of components built using 
-> [OverReact's fluent-style](#fluent-style-component-consumption).
-> 
-> _We have a long-term goal of writing our own formatter that extends from `dartfmt` to make it possible for our 
-> fluent-style to remain readable even after the formatter executes._
+#### dart_style
```

Comment:
@greglittlefield-wf I'd say - to make it look less important - just use bold text instead of a heading?